ethereum/dex/trades/insert_clipper_v2.sql

```diff
@@ -0,0 +1,126 @@
+CREATE OR REPLACE FUNCTION dex.insert_clipper_v2(start_ts timestamptz, end_ts timestamptz=now(), start_block numeric=0, end_block numeric=9e18) RETURNS integer
+LANGUAGE plpgsql AS $function$
+DECLARE r integer;
+BEGIN
+WITH rows AS (
+    INSERT INTO dex.trades (
+        block_time,
+        token_a_symbol,
+        token_b_symbol,
+        token_a_amount,
+        token_b_amount,
+        project,
+        version,
+        category,
+        trader_a,
+        trader_b,
+        token_a_amount_raw,
+        token_b_amount_raw,
+        usd_amount,
+        token_a_address,
+        token_b_address,
+        exchange_contract_address,
+        tx_hash,
+        tx_from,
+        tx_to,
+        trace_address,
+        evt_index,
+        trade_id
+    )
+    SELECT
+        dexs.block_time,
+        pa.symbol AS token_a_symbol,
```

Comment:
for lines 32-35, you pull this data from prices, but in v1 script, it pulls from erc20 tokens table. do you want to keep consistency? i think it'll be more reliable too, as prices table can fall behind as it depends on data from outside API.

Reply:
Updated to pull symbol and decimals data from the erc20 tokens table.